transform/src/index.ts

```diff
@@ -44,6 +44,30 @@ export class JSONTransform extends Visitor {
     this.visitClassDeclaration(node);
   }
 
+  resolveType(type: string, source: Src): string {
```

Comment:
This should be a recursive lookup. For example,
`bar.ts`
```typescript
export type Type1 = string;
```

`foo.ts`
```typescript
import { Type1 } from "./bar";

export type Type2 = Type1;
```

`index.ts`
```typescript
import { Type2 } from "./foo";

@json
class TypeAlias {
  baz: Type2 = "";
}
```

Will not work since `Type2` will resolve to `Type1`, not `string`